.dockerfilelintrc

```diff
@@ -0,0 +1,3 @@
+rules:
+  apt-get_missing_rm: off
+  apt-get_recommends: off
```

Comment:
seems ok, but is there a reason for disabling `apt-get_recommends` as well?

Reply:
Yes, it breaks the product.  I spent half a day figuring out how to get the Python dev dependencies added and then found that a bunch of SSL cert' dependencies went missing. Rather than spending days sleuthing that second problem it' wa far easier simply to suppress the dockerfilelint finding.  The only downside is that the generated image might be a little bigger than it otherwise would, but that does not impact our product.